src/nvim/path.c

```diff
@@ -1749,12 +1749,22 @@ char_u *find_file_name_in_path(char_u *ptr, size_t len, int options, long count,
   return file_name;
 }
 
-// Check if the "://" of a URL is at the pointer, return URL_SLASH.
+bool path_starts_with_windows_drive_letter(const char *p) {
```

Comment:
Looked around and indeed seems like there is no such existing function! Let's see if we can make it less verbose.

```suggestion
/** Checks for a Windows drive letter ("C:/") at the start of the path. */
bool path_has_drive_letter(const char *p) {
```